R/blpAuthenticate.R

```diff
@@ -64,22 +66,30 @@ blpAuthenticate <- function(uuid=getOption("blpUUID", NULL),
                             host=getOption("blpLoginHostname", "localhost"),
                             ip.address=getOption("blpLoginIP", NULL),
                             con=defaultConnection(),
-                            default=TRUE) {
+                            default=TRUE,
+                            isEmrsId = getOption("isEmrsId", FALSE)) {
```

Comment:
Nit: I personally like camelCase but we seem to have moved away from it.  Should be use `is_emrs_id` or `is_emrsId` ?

Reply:
happy to go with whatever works - seems like depending on the function there could be camelCase dot.separated or snake_case. Maybe dot.separated is most appropriate since the only other par in the function that it would apply to is ip.address, so could do is.emrs.id.

Reply:
It is arguably worse because we also mix all styles with the options :)    So not even sure what is best -- maybe leave it for the (user-facing) option?